### DISCUSSION TITLE
RPCS3: How to package/start a PSN game instead of an ISO dumped ROM?

### DISCUSSION THREAD

**Discussion by mczero80** [ruanformigoni/gameimage#137] [Q&A]
Title says it all. 

Gameimage starts dumped ISOs no problem, but there is no way to start a PSN game, which is essentially a pkg file that you have to install from RPCS3. Game shows up when testing RPCS3, but Gameimage wants to start a dumped game from a folder.

**Comment by ruanformigoni** [comment]
Hi @mczero80 , thanks for the info! I'll work on better integration with `.pkg` files for the next release.

**Comment by mczero80** [comment]
Thank you very much.

I have another idea: to make it possible to open the RPCS3 gui from the finished gameimage.
Example use case: changing the user name for multiplayer games (RPCN option in RPCS3).
